cmd/internal/planetscale_edge_database.go

```diff
@@ -189,9 +195,12 @@ func (p PlanetScaleEdgeDatabase) Read(ctx context.Context, w io.Writer, ps Plane
 		if lcErr != nil {
 			return currentSerializedCursor, errors.Wrap(err, "Unable to get latest cursor position")
 		}
+		if latestCursorPosition == "" {
```

Comment:
Necessary because if we get an empty stop position, the sync will run indefinitely.

Reply:
how would we get an empty stop position? from the user setting that or from VStream sending it somehow?

Reply:
When I tested locally (e2e), there was one case where I got an empty string in an error case. I forgot the exact conditions but thought it would be good to handle.